draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.scss

```diff
@@ -50,12 +50,15 @@ $ca-banner-collapse-height-delayed: margin-top $kz-var-animation-duration-fast
 }
 
 .iconWrapper {
+  width: 155px;
+  height: 155px;
```

Comment:
Where are these hard-coded values coming from? If these can be derived from another variable we should... if not, a comment to explain what 155px represents would be helpful

Reply:
I now see it was hard-coded in the HTML previously. A comment or a variable name would still be nice to make it easier to follow.

I also wonder if it should be called `.illustration` instead of `.iconWrapper` and then you could have `$illustrationSize: 155px`.

Reply:
Updated.